source/elements/oneMKL/source/architecture/other_architecture.inc.rst

```diff
@@ -11,10 +11,40 @@ This section covers all other features in the design of oneMKL architecture.
 
 .. _onemkl_spec_current_version:
 
-Current Version of this oneMKL Specification
+Specification Version and Compliance
 +++++++++++++++++++++++++++++++++++++++++++++
 
-This is the oneMKL specification which is part of the oneAPI specification version 1.0.0.
+Each oneMKL domain must define a preprocessor macro to represent the version of the specification that the implementation is compliant with.
+
+The macros for each domain are listed as follows:
+
+  | ONEMKL_BLAS_SPEC_VERSION
+  | ONEMKL_LAPACK_SPEC_VERSION
+  | ONEMKL_SPARSE_SPEC_VERSION
+  | ONEMKL_DFT_SPEC_VERSION
+  | ONEMKL_RNG_SPEC_VERSION
+  | ONEMKL_VM_SPEC_VERSION
+
+The specification version can be created by appending all digits of the specification version in the format of <MAJOR><MINOR>. MINOR version always uses two digits. This version can be used to check the compatibility of the implementation with the specification version.
+
+Version Example
+
+  | oneAPI 1.1 rev 1 will be represented as a numerical value of 101
+  | oneAPI 1.2 rev 1 will be represented as a numerical value of 102
+  | oneAPI 1.2 rev 2 will be represented as a numerical value of 102
+
+Macro Example
+
+.. code-block:: c
+
+  // For oneAPI 1.2 rev 1
+  #define ONEMKL_BLAS_SPEC_VERSION 102
+
+  // For oneAPI 1.2 rev 2
```

Comment:
Should we mention that `rev 2` is assuming to be the latest one since it's not present in the version macro?

Reply:
Added a line